host/docker/scripts/aic

```diff
@@ -53,6 +53,16 @@ function check_data_img {
     fi
 }
 
+function check_verity_metadata {
+    if [ -n "$(sudo find /var/lib/docker -name verity_metadata)" ]; then
```

Comment:
yes, but currently i don't find a better solution for this. 
since the dm-verity flag is decided at the build time, while the aic start scirpt can't get the status of the dm-verity(enabled/disabled).   or could we just use a roughly way that "always sleep 1 sec" before start the android container whether dm-verity enabled/disabled...

do you have any good idea about it?